packages/utils/browser/accept-values.ts

```diff
@@ -0,0 +1,49 @@
+import Bowser from 'bowser'
+
+// This code should not have been written, but there is no other way to get the default accept values for the browser 😣🤮
+// Values where taken from https://developer.mozilla.org/en-US/docs/Web/HTTP/Content_negotiation/List_of_default_Accept_values
+
+export type ACCEPT_CONTEXT = 'default' | 'image' | 'video' | 'audio'
+export const BROWSER_TYPES = ['Chrome', 'Firefox', 'Safari', 'Microsoft Edge', 'Opera']
```

Comment:
This just covers the browsers that where in the list from Mozilla, the goal of this PR is not complete coverage, just an enhancement as this is all very JS2010, where you hacked your way to a solution.

If the browser is not on the list it will fallback to undefined which will cause the api to fallback to a jpg.